packages/amplify_authenticator/lib/src/widgets/buttons.dart

```diff
@@ -102,23 +102,30 @@ class GoToSignUpButton extends StatelessWidget {
   Widget build(BuildContext context) {
     final SignInViewModel _authModelView =
         InheritedAuthViewModel.of(context)!.signInViewModel;
-    return Row(
-      children: [
-        const Text("No account? ",
-            style: TextStyle(
-              fontSize: AuthenticatorButtonConstants.fontSize,
-            )),
-        TextButton(
-          key: const Key(keyGoToSignUpButton),
-          onPressed: _authModelView.goToSignUp,
-          child: Text("Create account",
-              style: TextStyle(
+    return Padding(
+      padding: const EdgeInsets.only(top: 4),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          const Expanded(
+            child: const Text("No account?",
+                style: TextStyle(
+                  color: AuthenticatorButtonConstants.textColor,
                   fontSize: AuthenticatorButtonConstants.fontSize,
-                  color: Theme.of(context).primaryColor != Colors.blue
-                      ? Theme.of(context).primaryColor
-                      : AuthenticatorColors.primary)),
-        )
-      ],
+                )),
+          ),
+          TextButton(
+            key: const Key(keyGoToSignUpButton),
+            onPressed: _authModelView.goToSignUp,
+            child: Text("Create account",
+                style: TextStyle(
+                    fontSize: AuthenticatorButtonConstants.fontSize,
+                    color: Theme.of(context).primaryColor != Colors.blue
+                        ? Theme.of(context).primaryColor
+                        : AuthenticatorColors.primary)),
```

Comment:
@haverchuck We will need to update this at some point but it's not a huge deal right now